api/test/logs/logger_test.cc

```diff
@@ -70,17 +103,14 @@ class TestProvider : public LoggerProvider
 
 TEST(Logger, PushLoggerImplementation)
 {
-  // Push the new loggerprovider class into the global singleton
+  // Push the new loggerprovider class into the API
   auto test_provider = shared_ptr<LoggerProvider>(new TestProvider());
   Provider::SetLoggerProvider(test_provider);
 
   auto lp = Provider::GetLoggerProvider();
 
-  // GetLogger(name, options) function
+  // Get a logger instance and check whether it's GetName() method returns
+  // "test logger" as defined in the custom implementation
   auto logger = lp->GetLogger("TestLogger");
-
-  // GetLogger(name, args) function
-  std::array<string_view, 1> sv{"string"};
-  span<string_view> args{sv};
-  auto logger2 = lp->GetLogger("TestLogger2", args);
+  ASSERT_EQ("test logger", logger->GetName());
```

Comment:
Add `ASSERT_EQ(logger2->GetName(), "TestLogger2")`

Reply:
The API implementation returns a noop instance, so the name would always be "noop logger" which is what the first test checks for